tests/support/util.tcl

```diff
@@ -688,18 +688,18 @@ proc process_is_alive pid {
 proc pause_process pid {
     exec kill -SIGSTOP $pid
     wait_for_condition 50 100 {
-        [string match {*T*} [lindex [exec ps j $pid] 16]]
+        [string match {*T*} [lindex [exec ps -l -p $pid] 15]]
```

Comment:
the test failed on Macos: https://github.com/sundb/redis/actions/runs/17720225812/job/50351045472
maybe we can make another if check just for sunos.